### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Antanas**
Add homework task description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+
+Implementation of all the [required] lines is necessary for passing the test. Implementation of the non-required lines gives bonus points. Feel free to come up with and implement your own ideas in addition to, or instead of, the optional requirements.
+
+While working on this assignment, your are welcome to have fun and show off your strengths.
+
+## Required requirements
+- [ ] Allow the user to select multiple `.csv` files (file example is shown below)
+- [ ] List all selected files together with their filenames and line counts
+- [ ] Add an option to remove any file from the list
+- [ ] Show the average of age of all the users
```
Comment (ignas-gustainis) [inline README.md]: There is an immediate switch of context that makes this a bit confusing - users were never mentioned before in the task nor that that's what will appear in the input file.

We can either:
Switch the order of lines 13 and 14, specifically mention `that collects users from .csv files` in the latter.
or
Show the .csv file before the requirements, i.e.:
```The system will allow uploading .csv files of this format:
<example>
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+```
+curl -X POST https://frontend-homework.getsandbox.com/users
+   -H 'Content-Type: application/json'
+   -d '{"users":["David Bowie","Jimmy Hendrix","Elvis Presley"]}'
+```
+Note: api randomly fails (on purpose) with 500
+
+## Requirements
+### Required 
+- [ ] Allow the user to select multiple `.csv` files (file example is shown below)
```
Comment (Leden) [inline README.md]: above

File: README.md [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+- [ ] List all selected files together with their filenames and line counts
+- [ ] Add an option to remove any file from the list
+- [ ] Show the average of age of all the users
+- [ ] Add a "Submit" button that collects users from files and sends them to api (more info below)
```
Comment (Leden) [inline README.md]: above

**Comment by Leden** [comment]
Front ent?

**Pull request merged**